src/Dapr.PluggableComponents.AspNetCore/Resources.Designer.cs

```diff
@@ -0,0 +1,135 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
```

Comment:
Can this be generated during build and removed entirely from the repo?

Reply:
Historically, generated resource source files are committed to the repo.  This is because they're generated by the tooling when the corresponding `.resx` is *modified* and not during the actual *build*.